app/models/messages/XSDFile.scala

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Do you think the XSDFile is in the right place? It's within messages directory but isn't a message.

Reply:
Agreed, moved outside of messages directory.